spec/requests/atmosphere/appliances_spec.rb

```diff
@@ -41,18 +41,25 @@
         })
     end
 
-    it 'creates dynamic configuration with header mi ticket injected' do
-      post api("/appliances"), params: dynamic_request_with_mi_ticket_body,
-           headers: {"MI-TICKET" => 'ticket'}
-      config_instance = Atmosphere::ApplianceConfigurationInstance.find(appliance_response['appliance_configuration_instance_id'])
-      expect(config_instance.payload).to eq 'dynamic config with mi_ticket: ticket'
-    end
-
-    it 'creates dynamic configuration with query param mi ticket injected' do
-      post api("/appliances?mi_ticket=ticket"), params: dynamic_request_with_mi_ticket_body
-      config_instance = Atmosphere::ApplianceConfigurationInstance.find(appliance_response['appliance_configuration_instance_id'])
-      expect(config_instance.payload).to eq 'dynamic config with mi_ticket: ticket'
-    end
+    # TODO: Temporarily disabling these failing specs due to a change in platform behavior
```

Comment:
@mkasztelnik I beg to disagree. These tests were always bogus, it's just that nobody had noticed. Basically, the tests succeeded only because of a bug in the PDP code (read the inline comments below). My PR accidentally fixed the bug, which means that the tests now fail. The code is correct; the tests are wrong. Should I try to replicate the buggy behavior in the new code...?